src/Pages/Apply/DashBoard/DashBoard.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import ApplyList from './ApplyList';
+import styled from "styled-components";
+
+function DashBoard(props) {
+
+  const sumValue = (el) => {
+    const data = el.process
+    let objList = [];
+    for(let key in data){
+      let obj = {};
+      obj['name'] = key;
+      obj['value'] = data[key];
+      objList.push(obj);
+    }
+    return objList;
+  }
```

Comment:
- 지난 리뷰 내용에 언급했던 내용으로 기억합니다. 객체 key 자리에 변수 등을 쓸 것이 아니라면 dot notation으로 쓰는게 좋습니다.
- 로직을 다시 살펴보겠습니다. 객체를 받고 value를 순회하며 새로운 배열로 리턴하고 있습니다. 이는 다음과 같이 다시 써줄 수 있습니다.
```suggestion
  const sumValue = ({ process }) => Object.values(process)
```

Reply:
맞습니다.. 이걸 푸시한 다음에 이전리뷰를 봤기 때문에 지금 수정..(코쓱)

Reply:
라이프사이클 때문에 주었던 조건인데 데이터가 잘 들어오고 있기때문에 수정하였습니다. 👍